1-js/02-first-steps/11-logical-operators/9-check-login/ifelse_task.svg

```diff
@@ -1 +1,206 @@
l-rule="nonzero" d="M131.691 131.367v3l-59.302-.001v85.181h8l-9.5 19-9.5-19h8v-88.18h62.302z"/><path id="Path-1218-Copy-2" fill="#EE6B47" fill-rule="nonzero" d="M281.691 326.367v3l-59.307-.001-.319 90.139 8 .029-9.567 18.966-9.433-19.033 8 .027.324-91.633.005-1.494h62.297z"/><g id="Rectangle-354-+-Каково-“официальное”" transform="translate(127 81)"><path id="Rectangle-354" fill="#FFF9EB" stroke="#E8C48E" stroke-width="2" d="M2.06 52l91.55 50.856L185.161 52 93.611 1.144 2.058 52z"/><text id="Who's-there?" fill="#8A704D" font-family="OpenSans-Regular, Open Sans" font-size="14" font-weight="normal"><tspan x="51.722" y="55">Who&apos;s there?</tspan></text></g><g id="Rectangle-354-+-Каково-“официальное”-Copy" transform="translate(277 276)"><path id="Rectangle-354" fill="#FFF9EB" stroke="#E8C48E" stroke-width="2" d="M2.06 52l91.55 50.856L185.161 52 93.611 1.144 2.058 52z"/><text id="Password?" fill="#8A704D" font-family="OpenSans-Regular, Open Sans" font-size="14" font-weight="normal"><tspan x="59.03" y="55">Password?</tspan></text></g><g id="Rectangle-356-+-Отмена" transform="translate(39 191)"><path id="Rectangle-356" fill="#FFF" d="M0 2h60v20H0z"/><text id="Cancel" fill="#8A704D" font-family="OpenSans-Regular, Open Sans" font-size="14" font-weight="normal"><tspan x="7.5" y="15">Cancel</tspan></text></g><g id="Rectangle-356-+-Отмена-Copy" transform="translate(189 384)"><path id="Rectangle-356" fill="#FFF" d="M0 2h60v20H0z"/><text id="Cancel" fill="#8A704D" font-family="OpenSans-Regular, Open Sans" font-size="14" font-weight="normal"><tspan x="7.5" y="15">Cancel</tspan></text></g><g id="Rectangle-356-Copy-+-“Админ”" transform="translate(341 191)"><path id="Rectangle-356-Copy" fill="#FFF" d="M0 2h60v20H0z"/><text id="Admin" fill="#8A704D" font-family="OpenSans-Regular, Open Sans" font-size="14" font-weight="normal"><tspan x="7.703" y="15">Admin</tspan></text></g><g id="Rectangle-356-Copy-+-“Админ”-Copy" transform="translate(493 384)"><path id="Rectangle-356-Copy" fill="#FFF" d="M7 2h60v20H7z"/><text id="TheMaster" fill="#8A704D" font-family="OpenSans-Regular, Open Sans" font-size="14" font-weight="normal"><tspan x=".096" y="15">TheMaster</tspan></text></g><path id="Line" fill="#EE6B47" fill-rule="nonzero" d="M223 182v42.56h8l-9.5 19-9.5-19h8V182h3z"/><path id="Line-Copy-2" fill="#EE6B47" fill-rule="nonzero" d="M373 377v42.56h8l-9.5 19-9.5-19h8V377h3z"/><g id="Rectangle-356-Copy-2-+-“Админ”-Copy" transform="translate(191 191)"><path id="Rectangle-356-Copy-2" fill="#FFF" d="M0 2h60v20H0z"/><text id="Other" fill="#8A704D" font-family="OpenSans-Regular, Open Sans" font-size="14" font-weight="normal"><tspan x="9.996" y="15">Other</tspan></text></g><g id="Rectangle-356-Copy-2-+-“Админ”-Copy-Copy" transform="translate(341 384)"><path id="Rectangle-356-Copy-2" fill="#FFF" d="M0 2h60v20H0z"/><text id="Other" fill="#8A704D" font-family="OpenSans-Regular, Open Sans" font-size="14" font-weight="normal"><tspan x="9.996" y="15">Other</tspan></text></g></g></g></svg>
```

Comment:
Please restore the original SVG file.

Reply:
I don't see this file is restored. If there is something to translate in this SVG please take a look at the README.md file. You'll find some instructions on how to do that.